src/ui/buttons.html

```diff
@@ -6,9 +6,10 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head>
     <meta charset="utf-8" />
-    <title>Button Men Online &mdash; Buttons</title>
+    <title>Button Men Online</title>
```

Comment:
Hmm, this looks wrong at a glance - shouldn't it be "Buttons &mdash; Button Men Online"?